locales/en-US/formautofill.properties

```diff
@@ -2,32 +2,118 @@
+category.tel = phone
+category.email = email
 # LOCALIZATION NOTE (fieldNameSeparator): This is used as a separator between categories.
 fieldNameSeparator = ,\u0020
 # LOCALIZATION NOTE (phishingWarningMessage, phishingWarningMessage2): The warning
 # text that is displayed for informing users what categories are about to be filled.
 # "%S" will be replaced with a list generated from the pre-defined categories.
-# The text would be e.g. Also fill company, phone, email
+# The text would be e.g. Also autofills company, phone, email.
 phishingWarningMessage = Also autofills %S
 phishingWarningMessage2 = Autofills %S
+# LOCALIZATION NOTE (insecureFieldWarningDescription): %S is brandShortName. This string is used in drop down
+# suggestion when users try to autofill credit card on an insecure website (without https).
+insecureFieldWarningDescription = %S has detected an insecure site. Form Autofill is temporarily disabled
+
+# LOCALIZATION NOTE (autofillAddressesCheckbox): Label for the checkbox that enables autofilling addresses.
+autofillAddressesCheckbox = Autofill addresses
+# LOCALIZATION NOTE (learnMoreLabel): Label for the link that leads users to the Form Autofill SUMO page.
+learnMoreLabel = Learn more
+# LOCALIZATION NOTE (savedAddressesBtnLabel): Label for the button that opens a dialog that shows the
+# list of saved addresses.
+savedAddressesBtnLabel = Saved Addresses…
+# LOCALIZATION NOTE (autofillCreditCardsCheckbox): Label for the checkbox that enables autofilling credit cards.
+autofillCreditCardsCheckbox = Autofill credit cards
+# LOCALIZATION NOTE (savedCreditCardsBtnLabel): Label for the button that opens a dialog that shows the list
+# of saved credit cards.
+savedCreditCardsBtnLabel = Saved Credit Cards…
+
+# LOCALIZATION NOTE (manageAddressesTitle, manageCreditCardsTitle): The dialog title for the list of addresses or
+# credit cards in browser preferences.
+manageAddressesTitle = Saved Addresses
+manageCreditCardsTitle = Saved Credit Cards
+# LOCALIZATION NOTE (addressesListHeader, creditCardsListHeader): The header for the list of addresses or credit cards
+# in browser preferences.
+addressesListHeader = Addresses
+creditCardsListHeader = Credit Cards
+showCreditCardsBtnLabel = Show Credit Cards
+hideCreditCardsBtnLabel = Hide Credit Cards
+removeBtnLabel = Remove
+addBtnLabel = Add…
+editBtnLabel = Edit…
+
+# LOCALIZATION NOTE (addNewAddressTitle, editAddressTitle): The dialog title for creating or editing addresses
+# in browser preferences.
+addNewAddressTitle = Add New Address
+editAddressTitle = Edit Address
+givenName = First Name
+additionalName = Middle Name
+familyName = Last Name
+organization = Company
+streetAddress = Street Address
+city = City
+province = Province
+state = State
+postalCode = Postal Code
+zip = Zip Code
+country = Country or Region
+tel = Phone
+email = Email
+cancelBtnLabel = Cancel
+saveBtnLabel = Save
+countryWarningMessage = Form Autofill is currently available only for US addresses
```

Comment:
Is this still true if we're enabling localization?

Reply:
Yes, but we are **[planning](https://docs.google.com/document/d/1cmS1IKxXCd6tvOS8nRD6NwaoB_IWL2zMTl5nF0pOLts/edit)** to support more countries. In addition to having localization ready, we want to make sure our heuristic algorithm works well in the countries we are supporting.